src/components/document/section-toolbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext, useState } from "react";
+import { clone } from "mobx-state-tree";
+import { AppConfigContext } from "../../app-config-context";
+import { IToolbarModel } from "../../models/stores/problem-configuration";
+import { ToolbarComponent } from "../toolbar";
+import { SectionModelType } from "src/models/curriculum/section";
```

Comment:
Should be relative path.
There is a vscode setting for 'import module specifier' that may help if these are getting auto-generated.